components/script/dom/servoparser/mod.rs

```diff
@@ -140,6 +144,72 @@ impl ServoParser {
         self.script_nesting_level.get()
     }
 
+    pub fn resume_with_pending_parsing_blocking_script(&self, script: &HTMLScriptElement) {
```

Comment:
Let's add a comment indicating that this is implementing the "otherwise" branch of the script tag branch of https://html.spec.whatwg.org/multipage/syntax.html#parsing-main-incdata .